docs/tooling.rst

```diff
@@ -395,10 +395,10 @@ Docker Image
 ============
 
 These tools, especially `kando` are meant to be invoked inside containers via
-Blueprints. Although suggest using the released image when possible, we've also
+Blueprints. Although we suggest using the released image when possible, we've also
 made it simple to add these tools to your container.
 
-The released image, `kanisterio/kanister-tools:0.21.0`, is hosted by
+The released image, `kanisterio/kanister-tools`, is hosted by
```

Comment:
```suggestion
The released image, `kanisterio/kanister-tools:|version|`, is hosted by
```

Reply:
Unfortunately, this isn't rendering. I will look into why but likely because anything within quotes here is not interpreted. Another change I have coming up will help.